app/src/main/java/org/simple/clinic/drugs/selection/entry/confirmremovedialog/ConfirmRemovePrescriptionDialog.kt

```diff
@@ -38,6 +39,12 @@ class ConfirmRemovePrescriptionDialog : AppCompatDialogFragment(), UiActions {
     requireArguments().getSerializable(KEY_PRESCRIPTION_UUID) as UUID
   }
 
+  private val events by unsafeLazy {
+    removeClicks()
+        .compose(ReportAnalyticsEvents())
```

Comment:
When we add it here, we will also have to remove it from the corresponding controller, otherwise the analytics events will get reported twice.

Since you are on leave, I will make this correction.